src/index.ts

```diff
@@ -70,14 +70,18 @@ async function main() {
   });
 
   config.browser.userAgent = await browser.userAgent();
+  const selectedStores = config.store.stores.map(store => store.name);
```

Comment:
There's an active PR trying to accomplish this same thing with some history you might be interested in: https://github.com/jef/streetmerchant/pull/2383

This may not be ready for prime-time due to some other use-cases and idiosyncracies with `config` usage. Feel free to solve it, but maybe split the PRs.

Reply:
Aah ha... I'll have to take a look at the web part and see how that's working. I'm sure something can be done to make this all be one happy family.

Reply:
I don't think anyone doubts that, and I think that's what is happening in that other PR. I'm sure they'd appreciate any insight you can offer from looking at it!